src/ontology/cl-edit.owl

```diff
@@ -30774,11 +30783,10 @@ EquivalentClasses(obo:CL_4030029 ObjectIntersectionOf(obo:CL_0000542 ObjectSomeV
 
 # Class: obo:CL_4030030 (peripheral blood lymphocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-9990-8331>) Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0003-2534-198X>) Annotation(oboInOwl:hasDbXref <https://sciencing.com/what-peripheral-blood-4672930.html>) Annotation(oboInOwl:hasDbXref "PMID:31092833") obo:IAO_0000115 obo:CL_4030030 "A blood lymphocyte located in the flowing, circulating blood of the body.")
-AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4030030 <https://orcid.org/0000-0002-1773-2692>)
```

Comment:
Is there a reason to remove the contributor in this PR?

Reply:
It was initially inadvertently added and removed in this PR whilst trying to resolve the QC errors. I'll be doing a larger edit to in a separate PR. If you prefer, I can add it back here, but will be removing in the subsequent PR.

Reply:
Ok, that's fine.